homeassistant/components/netgear/sensor.py

```diff
@@ -37,11 +39,86 @@
         key="ssid",
         name="ssid",
         entity_category=EntityCategory.DIAGNOSTIC,
+        icon="mdi:wifi-marker",
     ),
     "conn_ap_mac": SensorEntityDescription(
         key="conn_ap_mac",
         name="access point mac",
         entity_category=EntityCategory.DIAGNOSTIC,
+        icon="mdi:router-network",
+    ),
+}
+
+SENSOR_TRAFFIC_TYPES = {
```

Comment:
This could be a tuple instead since we don't need to look up the description.

Reply:
done